packages/schema/src/cli/config.ts

```diff
@@ -19,22 +19,22 @@ export let config: ConfigType = schema.parse({});
  * Loads and validates CLI configuration file.
  * @returns
  */
-export function loadConfig(filename: string) {
-    if (!fs.existsSync(filename)) {
-        return;
-    }
-
-    let content: unknown;
+export function loadConfig( filename: string ) {
     try {
-        content = JSON.parse(fs.readFileSync(filename, 'utf-8'));
-    } catch {
-        throw new CliError(`Config is not a valid JSON file: ${filename}`);
+        const fileData = fs.readFileSync( filename, `utf-8` );
```

Comment:
I see there're extra leading and trailing spaces within parentheses. Did this happen when the code is auto-formatted in VSCode? Not sure if there's any inconsistency in prettier settings between our settings.

Reply:
Yes it's VSCode doing that specific formatting. I have prettier running but that looks like there is a conflict going on

Reply:
Okay I have prettier taking precedence now, looks like that's resolved.